erpnext/accounts/report/accounts_receivable/accounts_receivable.py

```diff
@@ -1009,7 +1009,7 @@ def setup_ageing_columns(self):
 				"{range3}-{range4}".format(
 					range3=cint(self.filters["range3"]) + 1, range4=self.filters["range4"]
 				),
-				"{range4}-{above}".format(range4=cint(self.filters["range4"]) + 1, above=_("Above")),
+				_("{range4}-Above").format(range4=cint(self.filters["range4"]) + 1),
```

Comment:
Doing this way are more control to translation, example in spanish i would translate this as "90 días a más" or "mayor a 90 días"